### DISCUSSION TITLE
Most recommended artifact in Actions? (Closed)

### DISCUSSION THREAD

**Discussion by jadasse** [PCBox/PCBox#50] [Q&A] [answered]
I see many executables in the Actions tab of CMake, MSYS2, and CodeQL. Upon entering one of them, other than the recommended 32-bit Windows ODR executable, I see MinGW32, LLVM, Qt GUI, and I see lots more variations. My OS is a Windows 10 Pro x86_64. Which artifact is the most recommended for my case?

**Comment by jadasse** [comment]
After testing each build, 86Box-Qt-ODR-Dev-Windows-MINGW32 is the most compatible with its respective Jenkins recommended build

**Reply by bachimusprime** [reply, marked as answer]
Don't use the Dev builds, they have extra, more experimental features that are known to not work yet.
